HYUNSANG_OH/app.js

```diff
@@ -84,6 +115,84 @@ app.post("/post", async (req, res, next) => {
   res.status(201).json({ message: "postCreated" });
 });
 
+//좋아요 누르기
+app.post("/likes", async (req, res, next) => {
+  const { userId, postId } = req.body;
+  const likes = await appDataSource.query(
+    `
+    SELECT
+        id,
+        user_id,
+        post_id
```

Comment:
프로젝트 전반적으로 들여쓰기를 2칸으로 맞추고 계신 것 같으니 그것과 동일하게 진행해 주세요~
```suggestion
      id,
      user_id,
      post_id
```

Reply:
세심한 개발자로..얼른 성장하겠습니다!